src/actions/websocket.ts

```diff
@@ -138,15 +138,18 @@ export function doReconnect(now: number) {
             await dispatch(getMyTeamMembers());
             const currentTeamMembership = getCurrentTeamMembership(getState());
             if (currentTeamMembership) {
+                const currentChannel = getCurrentChannel(state);
                 const fethcResult = await dispatch(fetchMyChannelsAndMembers(currentTeamId));
                 const data = (fethcResult as any).data || null;
                 dispatch(loadProfilesForDirect());
 
                 if (data && data.channels && data.members) {
                     const channelStillExists = data.channels.find((c: Channel) => c.id === currentChannelId);
                     const stillMemberOfCurrentChannel = data.members.find((m: ChannelMembership) => m.channel_id === currentChannelId);
+                    const config = getConfig(getState());
```

Comment:
Shouldn't be used `state` variable here instead of `getState` again?

Reply:
As this is happening on reconnect, there is another event that gets triggered if the server version has changed here https://github.com/mattermost/mattermost-mobile/blob/master/app/init/global_event_handler.js#L247 So I want to make sure we are using the latest config.